internal/transport/http2_server.go

```diff
@@ -333,7 +334,7 @@ func NewServerTransport(conn net.Conn, config *ServerConfig) (_ ServerTransport,
 		t.loopy.ssGoAwayHandler = t.outgoingGoAwayHandler
 		if err := t.loopy.run(); err != nil {
 			if logger.V(logLevel) {
-				logger.Errorf("transport: loopyWriter.run returning. Err: %v", err)
+				logger.Infof("transport: loopyWriter exited. Closing connection. Err: %v", err)
```

Comment:
I think we use the term `transport` to mean a connection. If you want to distinguish between the `transport` type and the underlying `net.Conn`, maybe you can be a more explicit?